mongoexport/options.go

```diff
@@ -46,6 +46,9 @@ type OutputFormatOptions struct {
 
 	// NoHeaderLine, if set, will export CSV data without a list of field names at the first line.
 	NoHeaderLine bool `long:"noHeaderLine" description:"export CSV data without a list of field names at the first line"`
+
+	// JSONFormat specifies what extended JSON format to export (canonical or relaxed). Defaults to relaxed.
+	JSONFormat string `long:"jsonFormat" default:"relaxed" description:"the extended JSON format to output, either canonical or relaxed (defaults to 'relaxed')"`
```

Comment:
[nit] if you add a "value-name" similar to the other options, it will make the --help output a little nicer.

Reply:
done in new commit.